src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/RazorCodeActionContext.cs

```diff
@@ -3,16 +3,16 @@
 
 using System;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
 using Microsoft.CodeAnalysis.Text;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
 {
     internal sealed class RazorCodeActionContext
     {
         public RazorCodeActionContext(
-            CodeActionParams request,
+            RazorCodeActionParams request,
```

Comment:
Given you reset the "Range" to be "SelectionRange" at the endpoint layer for now i'd keep this as CodeActionParams so we don't start depending on VisualStudio specific aspects of the code action parameters.